x-pack/plugins/apm/server/lib/backends/get_throughput_charts_for_backend.ts

```diff
@@ -59,18 +63,11 @@ export async function getThroughputChartsForBackend({
       },
       aggs: {
         timeseries: {
-          date_histogram: getMetricsDateHistogramParams({
-            start: startWithOffset,
-            end: endWithOffset,
-            metricsInterval: 60,
-          }),
-          aggs: {
-            throughput: {
-              rate: {
-                field: SPAN_DESTINATION_SERVICE_RESPONSE_TIME_COUNT,
-                unit: 'minute',
-              },
-            },
+          date_histogram: {
+            field: '@timestamp',
+            fixed_interval: intervalString,
+            min_doc_count: 0,
+            extended_bounds: { min: startWithOffset, max: endWithOffset },
```

Comment:
I much prefer this over `getMetricsDateHistogramParams`. So much easier for me to read!